### ISSUE TITLE
"load a plugin designed for TiddlyWiki Classic" error

### ISSUE THREAD

**Issue by rcrath** [Jermolene/TiddlyWiki5#2282]
I am getting an error message of "load a plugin designed for TiddlyWiki Classic" followed by 

>Internal JavaScript Error
Well, this is embarrassing. It is recommended that you restart TiddlyWiki by refreshing your browser
Uncaught TypeError: $tw.wiki.old_parseTiddler is not a function

when I try to delete the draft tiddler.  or try to open a reference to the plugin to try and delete it.  I removed the actual tiddlers from the tiddlers directory, but there seem to still be references to the file.  Running Node.js version, installed tw5 version 5.1.11, updating from 5.1.9, which does not have this problem I think.

These are just leftover imports from my TW2 install so I am fine with deleting them, but can't.  THis was not a problem with nodejs TW5 until this version.  LMK if you need more info.

**Comment by rcrath** [comment]
tried installing [TWGuidesGenPlugins/plugins/tw2parser.tid](https://github.com/Drakor/TWGuidesGenPlugins/blob/master/plugins/tw2parser.tid) since it contained "tw.wiki.old_parseTiddler " in the code, but that did not work

**Comment by rcrath** [comment]
workaround:  It seems once that plugin was installed (although now I can't find it!) the delete tiddler shown [here](https://groups.google.com/forum/#!topic/tiddlywiki/U9GYlwVXp6M) was able to delete the offending tiddlers and now I get no errors.  BUt note that the code is not quite right.  for convenience, here is what actually worked (with the error messages for me:

```

<table>
<tr><th>Filter </th><td><$edit-text tiddler="$:/temp/filter"/><br />put in "[tag[systemConfig]]"</td></tr>
</table>


<$list filter={{$:/temp/filter!!text}}>


<$button message="tm-delete-tiddler" param={{!!title}} class="btn-invisible btn-mini">{{$:/core/images/delete-button}}</$button> <$link to={{!!title}}>{{!!title}}</$link> 
</$list>

````

THe tag "systemConfig"was the generic tag for plugins in TWclassic.  this still left some old help files and tw2 plugins that I had installed but deactivated and such to clean up.  I still get the same error messages, but the tiddlers do delete.

**Comment by buggyj** [comment]
hi @rcrath, 
the error "Uncaught TypeError: $tw.wiki.old_parseTiddler is not a function" is caused by using an incompatible version of a plugin with tw5.1.11. Can you provide the names of which plugins you have installed.

There is an official version or the tw2parser see:
http://tiddlywiki.com/plugins/tiddlywiki/tw2parser/

**Comment by rcrath** [comment]
Hi, @buggyj I was able to remove the offending plugins using the code above.  Because I was able to delete them with the workaround, I no longer have the names.  

Whenever I tried to just delete them in the normal fashion I would get the error message.  It seems TW5 should be able to gracefully delete them once they are identified rather than throwing an error and having to figure out a slightly hacky workaround.  

will replace the forked version with the official one.  thanks.
